File: library/src/main/java/com/jhworks/library/utils/CheckNullUtils.java
package com.jhworks.library.utils;

/**
 * <p>空对象检查 </p>
 *
 * @author jiahui
 *         date 2018/1/9
 */
public class CheckNullUtils {

    public static void check(Object o) {
        if (o == null) throw new RuntimeException(o + " can not null!!!");
    }
}

File: library/src/main/java/com/jhworks/library/utils/CheckNullUtils.kt
package com.jhworks.library.utils

/**
 *
 * 空对象检查
 *
 * @author jiahui
 * date 2018/1/9
 */
object CheckNullUtils {

    fun check(o: Any?, msg: String = "$o can not null!!!") {
        if (o == null) throw RuntimeException(msg)
    }

    fun <T> isListEmpty(list: MutableList<T>?): Boolean {
        if (list == null || list.isEmpty()) return true
        return false
    }
}